packages/preview-button/admin/src/components/PreviewLink/index.css

```
@@ -64,3 +64,6 @@
 img {
   max-inline-size: 100%;
 }
+.utrecht-html ol {
+  list-style: decimal;
```

Ik zou de CSS property zo precies mogelijk doen, als het mogelijk is.

```suggestion
  list-style-type: decimal;
```